.github/workflows/prepare_deployment_branch.yaml

```diff
@@ -27,13 +27,19 @@ jobs:
           echo "Branch name: \"${BRANCH_NAME}\""
 
       - name: "Create branch '${{ env.BRANCH_NAME }}' to contain the changes for the deployment on ${{ env.DEPLOYMENT_DATE }}"
+
         uses: JosiahSiegel/remote-branch-action@dbe7a2138eb064fbfdb980abee918091a7501fbe
+        ## emvaldes@hotmail.com (Replace) - uses: ./.github/actions/remote-branch-action
```

Comment:
corp email please

Reply:
I have updated the email address to reflect the Aquia corporate email.